aineko/models/base.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Base models for Aineko."""
+import datetime
+from typing import Union
+
+from pydantic import BaseModel, Field
+
+from aineko.config import AINEKO_CONFIG
+
+
+class MessageData(BaseModel):
```

Comment:
I'm not sure `MessageData` is a good name for this class. Especially since the Kafka implementation we use has a class named `Message`. Why not use `WrappedMessage`? We often use `wrapped_message = MessageData` in the code

Reply:
I initially chose WrappedMessage, but I felt like the name was not self-explanatory enough for users. But maybe MessageData is worse.

Reply:
Posted in Slack for feedback: https://aineko-dev.slack.com/archives/C05U15ATA2V/p1706450232466239